src/views/domain/_bulkSetContacts.php

```diff
@@ -8,7 +8,15 @@
 use yii\helpers\ArrayHelper;
 use yii\helpers\Html;
 
-$unchangeableZones = [];
+/**
+ * @var \yii\base\Model $model
+ * @var \yii\base\Model[] $models
+ */
+
+$unchangeableZones = array_map(
+    fn ($model) => $model->domain,
+    array_filter($models, fn ($model) => $model->isContactChangeable())
+);
```

Comment:
в `$unchangeableZones` должны остаться НЕ изменяемые зоны, а твой код делает наоборот

как на счёт вот так?

```suggestion
$unchangeableZones = array_filter(array_map(
    fn ($model) => $model->isContactChangeable() ? null : $model->domain,
    $models
));
```

надо оставить вариант который легче читается
не могу понять какой :)

Reply:
@SilverFire який легше читається?